Added CI Fast Example (#198)
harness/developer-hub: tutorials/build-code/ci-getting-started.md

![CI Getting Started](static/ci-tutorial-fast/cigettingstartedtemplates.png)

## Create your pipeline​

```mdx-code-block
import Tabs from '@theme/Tabs';
import TabItem from '@theme/TabItem';
```

1. Fork the repository [https://github.com/rustd/jhttp](https://github.com/rustd/jhttp) into your GitHub account.
2. Follow the **Get Started** wizard in Harness CI.
   ```mdx-code-block
   <Tabs>
     <TabItem value="newaccount" label="New account" default>
       If you are signing in to Harness for the first time, select the <strong>Continuous Integration</strong> module after your initial sign in. This brings you to the <strong>Get Started</strong> wizard.
     </TabItem>
     <TabItem value="existingaccount" label="Existing account">
       If you have an existing Harness account, either create a new project or select an existing project, and then select the <strong>Continuous Integration</strong> module. In the <strong>Project</strong> pane, expand the <strong>Project Setup</strong> menu, and then select <strong>Get Started</strong>.
     </TabItem>
   </Tabs>
   ```

> 3. If you are signing in to Harness for the first time, select the Continuous Integration module after your initial sign in. This brings you to the Get Started wizard.

Reviewer: Remove this step because it duplicates the text in the New account tab.

> 4. When you are prompted to select a repository, search for jhttp, select the repository that you forked in the earlier step, and then select Configure Pipeline.

Reviewer: ```suggestion
4. When you are prompted to select a repository, search for jhttp, select the repository that you forked in the earlier step, and then select **Configure Pipeline**.
```

> 5. Select Starter Pipeline, and then select Create Pipeline.

Reviewer: ```suggestion
5. Select **Starter Pipeline**, and then select **Create Pipeline**.
```

> 6. From the left pane, select Pipelines, and then select your jhttp pipeline from the list.

Reviewer: ```suggestion
6. From the left pane, select **Pipelines**, and then select your jhttp pipeline from the list.
```

> 7. Switch from the Visual view to the YAML view, and then select Edit YAML.

Reviewer: ```suggestion
7. Switch from the **Visual** view to the **YAML** view, and then select **Edit YAML**.
```

> 8. A starter pipeline is created with a single stage.

Reviewer: This should not be a numbered step because the user does not take any action. It should be a line below step 8.

9. Replace the sample step section with the following:
```
              - step:
                  type: Run
                  name: Build Java App
                  identifier: Build_Java_App
                  spec:
                    shell: Sh
                    command: |-
                      echo "Welcome to Harness CI"
                      mvn -B package --file pom.xml
```

> 10. Select Save in the YAML editor.

Reviewer: ```suggestion
10. In the YAML editor, select **Save**.
```

## Run your pipeline​

> 1. Run your pipeline​

Reviewer: ```suggestion
1. In the **Pipeline Studio**, select **Run**.​
```

> 2. In the Build Type field, select Git Branch, and then enter main in the Branch Name field.

Reviewer: ```suggestion
2. In the **Build Type** field, select **Git Branch**, and then enter **main** in the **Branch Name** field.
```

> 3. Select Run Pipeline.

Reviewer: ```suggestion
3. Select **Run Pipeline**.
```

## Next steps

> ### Build & test Apache Kafka 4x faster on Harness CI 

Reviewer: Remove this heading. It's unnecessary.

> Learn how you can build and test Apache Kafka 4x faster on Harness CI 

Reviewer: ```suggestion
Learn how you can build and test Apache Kafka four times faster on Harness CI. 
```
